pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 clingo = "^5.5.2"
-pandas = "^1.4.2"
+pandas = "1.4.2"
```

Comment:
This is fixing the pandas version and not the stubs. I don't think we want to be so restrictive if we can avoid it.

Reply:
I did it because of this 
<img width="812" alt="image" src="https://user-images.githubusercontent.com/35775163/176828121-7079c936-a13c-4d97-9545-e9aaed2f0aaa.png">
what should I do instead?

Reply:
I updated the lock file in https://github.com/cmudig/draco2/pull/330. Let's see whether it passes.